src/helpers/deprecation.ts

```diff
@@ -37,7 +37,7 @@ function deprecateQuery<QueryFn extends (...args: any) => any>(
 
 const warned: { [functionName: string]: boolean } = {};
 
-// istambul ignore next: Occasionally used
+/* istanbul ignore next: occasionally used */
```

Comment:
what is istanbul?

Reply:
This comment disable code coverage checking of the next line. Instanbul is the code cov used by Jest.